src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -45,20 +45,20 @@ static DISPLAY_TYPE gClipHeightDisplayType = DISPLAY_TYPE::DISPLAY_UNITS;
 
 #pragma region Widgets
 
+static constexpr const rct_string_id WINDOW_TITLE = STR_VIEW_CLIPPING_TITLE;
```

Comment:
Why did you define a constant for this, seeing as this is only used once?

Reply:
Well, I took that to be the convention. All windows that use the `WINDOW_SHIM` macro have a `WINDOW_TITLE` constant defined, unless there are several tabs involved. (I used `grep -RE \\bWINDOW_TITLE\\b *.cpp src/openrct2-ui/windows` to confirm.)

Reply:
It is mostly a remnant from when the original author of the window shim wrote it - initially, you had to #define the WINDOW_TITLE in the window definition file and this would then be picked up by the window shim #define. IOW, it would act like a global variable. I changed WINDOW_SHIM to simply take variables instead, but I didn't bother changing all the files. So it's not really a convention.

This can be left as-is, btw.

Reply:
Ah, alright, thanks for the clarification. Perhaps we can simplify it eventually, then.